src/sentry/mediators/service_hooks/creator.py

```diff
@@ -44,12 +45,17 @@ def call(self):
 
     def _create_service_hook(self):
         application_id = self.application.id if self.application else None
-
-        return ServiceHook.objects.create(
+        # there must be at least one project
+        project_id = self.projects[0].id
```

Comment:
Could you hit an IndexError, do you need to handle it?

Reply:
This shouldn't ever happen since the sentry app installation checks for projects and doesn't call the service hook creator and the service hook endpoint is specific to a project so there'd always need to be at least one